### PR TITLE
Mellow withdrawal testnet qa fixes

### PR DESCRIPTION
**Pull request by 0xZenus**

### CODE REVIEW

File: src/entities/mellow/mellowLpRouter.ts [hunk 1]
```diff
@@ -239,6 +237,26 @@ class MellowLpRouter {
     return this.userIndividualCommittedDeposits.reduce((total, deposit) => total + deposit, 0);
   }
 
+  public get userPendingDeposit(): number {
+    return this.userIndividualPendingDeposit.reduce((total, deposit) => total + deposit, 0);
+  }
+
+  public get userIndividualDeposits(): number[] {
+    if (
+      !(this.userIndividualPendingDeposit.length === this.userIndividualCommittedDeposits.length)
```
Comment (avniculae) [inline src/entities/mellow/mellowLpRouter.ts:246]: why not !== instead of !(...==...)?

File: src/entities/mellow/mellowLpVault.ts [hunk 2]
```diff
@@ -208,10 +210,30 @@ class MellowLpVault {
...
     return this.userIndividualCommittedDeposits.reduce((total, deposit) => total + deposit, 0);
   }
 
+  public get userPendingDeposit(): number {
+    return this.userIndividualPendingDeposit.reduce((total, deposit) => total + deposit, 0);
+  }
+
+  public get userIndividualDeposits(): number[] {
+    if (
+      !(this.userIndividualPendingDeposit.length === this.userIndividualCommittedDeposits.length)
```
Comment (avniculae) [inline src/entities/mellow/mellowLpVault.ts:223]: same

File: src/entities/mellow/mellowLpRouter.ts [hunk 3]
```diff
@@ -239,6 +237,26 @@ class MellowLpRouter {
...
+      return [];
+    }
+
+    return this.userIndividualPendingDeposit.map(
+      (pendingDeposit, index) => pendingDeposit + this.userIndividualCommittedDeposits[index],
+    );
+  }
+
+  public get userDeposit(): number {
+    return this.userIndividualDeposits.reduce((total, deposit) => total + deposit, 0);
```
Comment (avniculae) [inline src/entities/mellow/mellowLpRouter.ts:257]: shouldn't it be `this.userIndividualDeposits()`?

File: src/entities/mellow/mellowLpVault.ts [hunk 4]
```diff
@@ -208,10 +210,30 @@ class MellowLpVault {
...
+      return [];
+    }
+
+    return this.userIndividualPendingDeposit.map(
+      (pendingDeposit, index) => pendingDeposit + this.userIndividualCommittedDeposits[index],
+    );
+  }
+
+  public get userDeposit(): number {
+    return this.userIndividualDeposits.reduce((total, deposit) => total + deposit, 0);
```
Comment (avniculae) [inline src/entities/mellow/mellowLpVault.ts:234]: same?

Comment (0xZenus) [reply src/entities/mellow/mellowLpVault.ts:234]: no, because it has ``get``

File: src/entities/mellow/mellowLpRouter.ts (hunk 1 above)
Comment (0xZenus) [reply src/entities/mellow/mellowLpRouter.ts:246]: hmm not strong reasons

File: src/entities/mellow/mellowLpVault.ts (hunk 2 above)
Comment (0xZenus) [reply src/entities/mellow/mellowLpVault.ts:223]: hmm not strong reasons

**Review by avniculae — APPROVED** [review]

**Pull request merged**